terraform/deployments/govuk-publishing-platform/app_router.tf

```
@@ -28,6 +28,7 @@ module "router" {
   source = "../../modules/app"
 
   service_name                     = "router"
+  backend_virtual_service_names    = [] # TODO what services does Router use?
```

I'm pretty sure router doesn't use any services (other than its DB, which we're not modelling as a mesh service yet).

```suggestion
  backend_virtual_service_names    = [] # Router doesn't use any other services
```

---

I've added frontend for now, since other than the DB, I think Router sends requests to many other apps registered as backends.

---

Oh yeah 🤦🏻